astropy/coordinates/tests/test_erfa_astrom.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
+import pytest
+
+
+def test_science_state():
+    import astropy.units as u
+    from astropy.coordinates.erfa_astrom import (
```

Comment:
Just move the imports up top (here and below)

Reply:
I made the experience that having imports in the test functions is a dramatic quality of life improvement when import breaks.

Having the imports inside the test will make the test fail instead of making the test collection fail.

This allows e.g. `pytest --last-failed` to work properly.

Reply:
I can see that being useful if the imports in all the tests are different, but that is not the case here... Also, this is something we try to do everywhere in astropy...

Reply:
Ok. But maybe something worth discussing, as a single failed import will stop the full test collection and then mess with the `--last-failed` and similar functions of pytest. I'll change here and maybe open an issue.

Reply:
done